Dockerfile

```diff
@@ -12,6 +12,5 @@ RUN make install
 ############# gardener-extension-shoot-oidc-service
 FROM gcr.io/distroless/static-debian11:nonroot AS gardener-extension-shoot-oidc-service
 
-COPY charts /charts
```

Comment:
I think the extension still needs the `charts/images.yaml` file so that it can work when no image vector overwrite configmap is provided?

Reply:
I think that this should be handled by the embedding done [here](https://github.com/gardener/gardener-extension-shoot-oidc-service/blob/9e06594e813055ad0d527eec4407007f2573bb9d/charts/images.go#L11) and the variable will be populated during compile time. I will test it just to be sure though.

Reply:
Good point, my mistake.
Thanks for the clarification!